### PR TITLE
Update 100-context.md

### PR DESCRIPTION
**Pull request by Yubraj977**
This article help me  a lot to grab the idea about how the context is solving the problem of the prop drilling. I think beginner should start with this article to grasp the knowledge about the problem that trying to solve by the the context

### CODE REVIEW

**Review by Mohitbishukarma — COMMENTED** [review]
That is nice article 👍👍

**Review by san-poon — COMMENTED** [review]
https://www.w3schools.com/react/react_usecontext.asp, this site just confuse the newcomers.

**Comment by Yubraj977** [comment]
> https://www.w3schools.com/react/react_usecontext.asp, this site just confuse the newcomers.

Maybe for you it created a confusions but for me this is the article that make me clearly understand what is the problem with normal props and when we should avoid prop drilling

File: src/data/roadmaps/react/content/106-state-management/100-context.md [hunk 1]
```diff
@@ -6,5 +6,7 @@ In a typical React application, data is passed top-down (parent to child) via pr
 
 Visit the following resources to learn more:
 
+- [Basic react use context ](https://www.w3schools.com/react/react_usecontext.asp)
```
Comment (dansholds) [inline src/data/roadmaps/react/content/106-state-management/100-context.md:9]: please remove the whitespace after context and change the title to "Basic useContext Guide" or something of that nature.

**Review by dansholds — CHANGES_REQUESTED** [review]

File: src/data/roadmaps/react/content/106-state-management/100-context.md [hunk 2]
```diff
@@ -6,5 +6,7 @@ In a typical React application, data is passed top-down (parent to child) via pr
...
 - [Passing Data Deeply with Context](https://react.dev/learn/passing-data-deeply-with-context)
 - [State with useContext and useState/useReducer](https://www.robinwieruch.de/react-state-usereducer-usestate-usecontext/)
+
```
Comment (kamranahmedse) [inline src/data/roadmaps/react/content/106-state-management/100-context.md:12]:
```suggestion
```

File: src/data/roadmaps/react/content/106-state-management/100-context.md [hunk 3]
```diff
@@ -6,5 +6,6 @@ In a typical React application, data is passed top-down (parent to child) via pr
 
 Visit the following resources to learn more:
 
+- [Basic react use context ](https://www.w3schools.com/react/react_usecontext.asp)
```
Comment (kamranahmedse) [inline src/data/roadmaps/react/content/106-state-management/100-context.md:9]:
```suggestion
- [Basic useContext Guide](https://www.w3schools.com/react/react_usecontext.asp)
```

**Pull request merged**